deploy/servctl_utils/kubectl_utils.py

```diff
@@ -0,0 +1,167 @@
+import logging
+import time
+
+from deploy.servctl_utils.shell_utils import run
```

Comment:
copy/pasted this file directly from hail_elasticsearch_pipelines where we were previously importing it from, and then changed this import to use out implementation of run instead of the one in the other repo. They are almost identical, except ours actually handles unicode errors properly. I made no other change to the file